AppHandling/Run-AlPipeline.ps1

```diff
@@ -1066,7 +1067,7 @@ if ($installOnlyReferencedApps) {
     $installTestApps += @($installApps)
 }
 
-$vsixFile = DetermineVsixFile -vsixFile $vsixFile
+$vsixFile = GetAlLanguageExtension -vsixFile $vsixFile -extract;

```

Comment:
Extracting the vsixFile here will disable the GitHub cache - on GitHub hosted runners, this will always download and extract the vsix file